sweet/benchmarks/cockroachdb/main.go

```diff
@@ -0,0 +1,611 @@
+// Copyright 2024 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !wasm
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/pkg/errors"
+	"golang.org/x/benchmarks/sweet/benchmarks/internal/driver"
+	"golang.org/x/benchmarks/sweet/benchmarks/internal/server"
+	"golang.org/x/benchmarks/sweet/common/diagnostics"
+	"os"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
+)
+
+const (
+	// Arbitrarily chosen to match the cockroachdb default.
+	basePort = 26257
+	// The percentage of memory to allocate to the cache.
```

Comment:
Nit: "...to allocate to the pebble cache."

Reply:
Done.